src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -210,6 +223,91 @@ def run(
                 )
                 raise unexpected_error from e
 
+    def copy_files_except(self, src_dir, dst_dir, exclude_file):
```

Comment:
Please move this function to utils folder or change name to` _copy_file_except`.